tests/test.util.js

```diff
@@ -63,7 +63,19 @@ describe('Utils', function() {
             // environments, so we need to redefine it whilst running these
             // tests.
             origNavigator = Object.getOwnPropertyDescriptor(window, "navigator");
+            if (origNavigator === undefined) {
+                // Object.getOwnPropertyDescriptor() doesn't work
+                // properly in any version of IE
+                this.skip();
+            }
+
             Object.defineProperty(window, "navigator", {value: {}});
+            if (window.navigator.languages !== undefined) {
+                // Object.defineProperty() doesn't work properly in old
+                // versions of Chrome
+                this.skip();
+            }
+
```

Comment:
Were you able to reproduce the issue locally with Chrome version <= 48? As it seems like saucelab's linux machines are limited to Chrome 48 which isn't very old (~Jan 2016). After a quick look I can't find any chrome bug w.r.t this

Anyway, since you are actually checking the value of languages, we should be fine either way.